hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/token/TokenAccessorImpl.java

```diff
@@ -218,12 +218,16 @@ public long balanceOf(Address account, Address token) {
 
     @Override
     public long staticAllowanceOf(final Address owner, final Address spender, final Address token) {
-        final var tokenAllowanceId = new AbstractTokenAllowance.Id();
-        tokenAllowanceId.setOwner(entityIdFromAccountAddress(owner));
-        tokenAllowanceId.setSpender(entityIdFromAccountAddress(spender));
-        tokenAllowanceId.setTokenId(entityIdNumFromEvmAddress(token));
+        if(properties.isAllowanceEnabled()) {
```

Comment:
Prefer to fail fast and keep the other code as unchanged.
```suggestion
        if (!properties.isAllowanceEnabled()) {
            throw new UnsupportedOperationException("allowance(address owner, address spender) is not supported.");
        }
```